app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -85,8 +86,8 @@ public function getAdditionalData(array $excludeAttr = [])
                 } elseif ($attribute->getFrontendInput() == 'price' && is_string($value)) {
                     $value = $this->priceCurrency->convertAndFormat($value);
                 }
-
-                if (is_string($value) && strlen($value)) {
+    
+                if ($value instanceof Phrase || (is_string($value) && strlen($value))) {
```

Comment:
I don't see a big difference between current solution and the one you proposed.
It's not mandatory to convert Phrase into string to pass data into template